Sources/ProjectDescription/Dependencies/CarthageDependencies.swift

```diff
@@ -5,20 +5,13 @@ public struct CarthageDependencies: Codable, Equatable {
     /// List of depedencies that will be installed using Carthage.
     public let dependencies: [Dependency]
 
-    /// List of options for Carthage installation.
-    public let options: Set<Options>
-
     /// Creates `CarthageDependencies` instance.
-    /// - Parameters:
-    ///   - dependencies: List of depedencies that can be installed using Carthage.
-    ///   - options: List of options for Carthage installation.
+    /// - Parameter dependencies: List of depedencies that can be installed using Carthage.
     public static func carthage(
```

Comment:
What about using the init instead? Given it will be passed to a `carthage` property, having to repeat the name is a bit strange, isn't it?
We would go from:
```
let dependencies = Dependencies(
    carthage: .carthage([
        …
    ]
)
)
```
to
```
let dependencies = Dependencies(
    carthage: .init(
        [
            …
        ]
    )
)
```

We could go even further, and make it `ExpressibleByArrayLiteral`, so it would be even clearer:
```
let dependencies = Dependencies(
    carthage: [
        …
    ],
    …
)
```

Reply:
> We could go even further, and make it ExpressibleByArrayLiteral, so it would be even clearer:

I believe this would be possible to do already. But yeah, I think the approach you have outlined might be more straightforward, so +1 from me.

Reply:
> What about using the init instead?

That was done on propose. The discussion about it can be found [here](https://github.com/tuist/tuist/pull/2532#issuecomment-781462676).

> We could go even further, and make it `ExpressibleByArrayLiteral`, so it would be even clearer:

👍

Reply:
Should we remove `static` builders and make `init` public?